spec/core_functions/color/same.hrx

```diff
@@ -0,0 +1,28 @@
+<===> known_space/input.scss
+a {b: same(hsl(0 100% 25.1%), 'red')}
+
+<===> known_space/output.scss
+a {b: false }
+
+================================================================================
```

Comment:
```suggestion
<===>
================================================================================
```

See the [spec structure](https://github.com/sass/sass-spec#spec-structure) on how to include an "end of  file line" for each test.